_episodes/interviews/2023-03-17-alexander-lopatin.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Cargo Limit with Alexander Lopatin"
+date: 2023-03-17T17:03:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-03-17-alexander-lopatin.mp3
+duration: "26:13"
+length: "25181486"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Alexander Lopatin](https://codonaft.com/), creator of [Cargo Limit](https://crates.io/crates/cargo-limit), a cargo plugin that will show errors in your Rust code before any warnings.
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+- [@0:00] - Alexander's programming background
+- [@3:17] - What made Alexander interested in the Rust
+- [@4:43] - What is Cargo Limit?
+- [@9:19] - Cargo Limit's features
+- [@11:20] - Improvements that Alexander wants to do on Cargo Limit
+- [@12:52] - Cargo extensions and installation
+- [@13:31] - Alexander's process of creating Cargo extensions, plugins, and fixing issues
```

Comment:
```suggestion
- [@13:31] - Alexander's process for creating Cargo extensions and plugins
```

Reply:
not a fan of the comma before the `and`? :D I tend to prefer that style, but will try to keep that in mind for the future.

Reply:
Oh, I'm a big fan of Oxford commas, but not when the list has only two items (like here)